File: backend/local-api-module/src/main/java/org/vyatsu/localApiModule/dto/response/api/ApiKeyDto.java
package org.vyatsu.localApiModule.dto.response.api;

import lombok.*;
import org.vyatsu.localApiModule.entity.enums.ApiKeyType;

import java.io.Serializable;
import java.time.LocalDateTime;


@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class ApiKeyDto implements Serializable {
    Long id;
    String key;
    String clientId;
    ApiKeyType type;
    Boolean isExpired;
    Long userId;
}

File: app/src/main/java/com/bytebusters/android/mpsystem/dto/response/api/ApiKeyDto.kt
package com.bytebusters.android.mpsystem.dto.response.api

import com.bytebusters.android.mpsystem.entity.enums.ApiKeyType
import java.io.Serializable

class ApiKeyDto : Serializable {
    var id: Long? = null
    var key: String? = null
    var type: ApiKeyType? = null
}